lms/djangoapps/bulk_email/tasks.py

```diff
@@ -794,6 +794,7 @@ def _submit_for_retry(entry_id, email_id, to_list, global_email_context,
         )
         raise retry_task
     except RetryTaskError as retry_error:
+        from nose.tools import set_trace; set_trace()
```

Comment:
This line is where the errors reported in splunk are coming from; they're supposedly errors raised during a retry. Problem being, the ratelimit errors never get this far without having already started a *different* retry attempt. So I have yet to actually hit this line in practice.

Maybe it has to do with using actual celery queues instead of inlining them like devstack...